src/Microsoft.Management.UI.Internal/ManagementList/Common/WpfHelp.cs

```diff
@@ -214,10 +214,7 @@ public static void AddChild(FrameworkElement parent, FrameworkElement element)
         {
             ArgumentNullException.ThrowIfNull(element);
 
-            if (parent == null)
-            {
-                throw new ArgumentNullException("element");
-            }
+            ArgumentNullException.ThrowIfNull(parent, nameof(element));
```

Comment:
@xtqqczze The second parameter appears to be incorrect.

Reply:
@iSazonov suggested this change https://github.com/PowerShell/PowerShell/pull/18792#discussion_r1048671618

Reply:
The code is frozen and we should keep its behavior unchanged.